src/prefer-Array-from.ts

```diff
@@ -0,0 +1,45 @@
+import { Rule } from "eslint";
+import { TSESTree } from "@typescript-eslint/experimental-utils";
+
+const arrayFromMapRule: Rule.RuleModule = {
```

Comment:
Why not?

```suggestion
export const arrayFromMapRule: Rule.RuleModule = {
```

Reply:
+1 here, exports at source make it easier to know if a value is exported as opposed to having to check the bottom of the file.

Reply:
thanks! will fix this together w/ tests